dcs-dtc/Models/FA18/FA18Configuration.cs

```diff
@@ -129,15 +129,9 @@ internal static FA18Configuration FromCombatFliteXML(FA18Configuration previous,
 
                 var lat = pos.Element("Latitude")?.Value;
                 var lon = pos.Element("Longitude")?.Value;
-                var dLat = double.Parse(lat.Replace('.', ','));
-                var dLon = double.Parse(lon.Replace('.', ','));
-                //if (
-                //    !double.TryParse(lat, out var dLat) ||
-                //    !double.TryParse(lon, out var dLon))
-                //{
-                //    continue;
-                //}
-                float.TryParse(pos.Element("Altitude")?.Value.Replace('.', ','), out var elevation);
+                var dLat = double.Parse(lat.Replace(',', '.'));
```

Comment:
Has this be verified with different locales? For example in german locale, the decimal point is replaced with a decimal comma.